website_rma/views/request_rma_form.xml

```diff
@@ -0,0 +1,174 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2020 Tecnativa - Ernesto Tejeda
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <data>
+        <record id="request_rma_page_view" model="ir.ui.view">
```

Comment:
Check the operations that need  `sudo` or a portal user will get a permissions error.